build.gradle

```diff
@@ -17,10 +17,8 @@ allprojects {
   group = 'com.google.api'
 }
 
-// Project names not used for release
-def nonReleaseProjects = ['benchmark']
 // Project names not using the default publication configuration
-def noDefaultPublications = ['benchmark', 'gax-bom']
+def noDefaultPublications = ['gax-bom']
 def nonJavaProjects = ['gax-bom']
```

Comment:
Now I think we can just remove `noDefaultPublications` and use `nonJavaProjects` instead where it's used. It's only for `gax-bom` now.